products/statement-generator/src/components/LinkAsText.tsx

```diff
@@ -1,26 +1,15 @@
 import React from 'react';
 import { makeStyles, createStyles, Link } from '@material-ui/core';
 
-const useStyles = makeStyles(() =>
+const useStyles = makeStyles(({ palette }) =>
   createStyles({
     linkAsText: {
-      color: '#25003F',
-      '&:visited': {
-        color: '#25003F', // Same color for visited links
-      },
-      '&:hover': {
-        color: '#25003F', // Same color on hover
-      },
+      color: palette.common.black,
     },
   })
 );
 
-interface LinkAsTextProps {
-  link: string;
-  label?: string;
-}
-
-export default function LinkAsText({ link, label }: LinkAsTextProps) {
+export default function LinkAsText({ link }: any) {
```

Comment:
rather than using `any` as the type, can you make this a 'string' type?

or bring back the LinkAsTextProps interface and update it ?